app/build.gradle

```diff
@@ -42,6 +42,10 @@ dependencies {
     implementation "org.apereo.cas:cas-server-core-configuration-metadata-repository:${casVersion}"
     implementation "org.apereo.cas:cas-server-core-api-configuration-model:${casVersion}"
 
+    // temporarily added for CAS 6.4.0-RC5 to override snapshot dependency
+    implementation "org.apereo.service.persondir:person-directory-impl:2.0.9"
```

Comment:
This is not a good idea. The change will get removed in about one week. If you do need to work RC5, you will have apply the change manually for now until RC6 is released.

Reply:
Yeah I knew it was temporary, I just have a sample project that was running the initializr by cloning this repo in CI so it wasn't working but I could point it at tag or at the initializr running in heroku.